static/app/components/stream/group.tsx

```diff
@@ -639,7 +639,6 @@ const Wrapper = styled(PanelItem)<{
         height: 100%;
         background-color: ${p.theme.bodyBackground};
         opacity: 0.4;
-        z-index: 1;
```

Comment:
Any idea why it was here to begin with?

Reply:
It was added as part of a big css block [here](https://github.com/getsentry/sentry/pull/23386/files#diff-bb196d099513ec8850092ea52e6f5359b6e199b0ac7878dd9e9dcdd5e377841cR515), just an oversight I think.